plugins/woocommerce-blocks/assets/js/blocks/order-confirmation/additional-fields-wrapper/block.json

```diff
@@ -0,0 +1,29 @@
+{
+	"name": "woocommerce/order-confirmation-additional-fields-wrapper",
+	"version": "1.0.0",
+	"title": "Additional Checkout Fields Section",
+	"description": "Display the additional fields section.",
```

Comment:
Should we describe in more detail what this is? i.e. that it is for additional checkout fields that are rendered in the 'contact' or 'address' location.